### PR TITLE
Fix a few problems I encountered during dogfooding

### PR DESCRIPTION
**Pull request by firtina-improbable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Each commit to fix another problem
#### Tests
Testing with an external project
#### Documentation
Internal fixes

### CODE REVIEW

File: workers/unity/Packages/com.improbable.gdk.tools/Common.cs [hunk 1]
```diff
@@ -67,7 +67,14 @@ internal static string GetThisPackagePath()
             }
 
             path = path.Replace("file:", string.Empty);
-            path = $"{PackagesDir}/{path}";
+
+            if (Path.IsPathRooted(path))
+            {
+                // A "rooted path" is an absolute path, therefore it will should point directly at the package.
```
Comment (jared-improbable) [inline workers/unity/Packages/com.improbable.gdk.tools/Common.cs]: Remove "should"

Comment (firtina-improbable) [reply workers/unity/Packages/com.improbable.gdk.tools/Common.cs]: 👍

**Comment by firtina-improbable** [comment]
doing a local sanity test with clean project just in case

**Comment by firtina-improbable** [comment]
sanity test passed, things still work from clean state

**Pull request merged**